src/dataspec/base.py

```diff
@@ -442,9 +512,6 @@ def validate(self, v) -> Iterator[ErrorDetails]:
         for i, e in enumerate(v):
             yield from _enrich_errors(self._spec.validate(e), self.tag, i)
 
-    def _default_conform(self, v):
-        return (self._out_type or type(v))(self._spec.conform(e) for e in v)
```

Comment:
Is `_out_type` used anymore? Should it be removed from `CollSpec`?

Reply:
I suppose I could remove it. I left it there just so you could introspect the object and see what output type was given. I slightly favor keeping it, however.